bigquery/samples/export_data_to_cloud_storage.py

```diff
@@ -11,16 +11,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import argparse
 import uuid
 
-from bigquery.samples.utils import get_service
-from bigquery.samples.utils import poll_job
-from six.moves import input
+from utils import get_service, poll_job
 
 
 # [START export_table]
 def export_table(service, cloud_storage_path,
-                 projectId, datasetId, tableId,
+                 project_id, dataset_id, table_id,
```

Comment:
True, but I'd rather follow python convention where possible:

> Names that are visible to the user as public parts of the API should follow conventions that reflect usage rather than implementation.
> ...
> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility.